src/server/api/routes/api-router.js

```diff
@@ -35,4 +29,40 @@ router.use('/batch', getBatchRouter);
 router.use('/batch-default-values', batchDefaultValuesRouter);
 router.use('/delete-specific-batch', deleteSpecificBatchDataRouter);
 
+// swagger-ui-express
+// const swaggerDocument = require('../../config/swagger.json');
+// const swaggerUi = require('swagger-ui-express');
+const swaggerJsDoc = require('swagger-jsdoc');
+const swaggerUi = require('swagger-ui-express');
+
+const swaggerOptions = {
+  swaggerDefinition: {
+    info: {
+      version: '1.0',
+      title: 'Final project',
+      description: 'API documentation for the final project',
+      contact: {},
+    },
+    host: '',
+    basePath: '/api',
+  },
+  securityDefinitions: {
+    firebase_auth: {
+      type: 'oauth2',
+      authorizationUrl: 'seasony-login.firebaseapp.com',
+      description: 'This API uses OAuth 2.',
+      flows: 'authorizationCode',
+      scopes: {
+        read: 'read data based on user uid',
+        write: 'write data based on user uid',
+      },
+    },
+  },
+  apis: ['./src/server/api/routes/*.js'],
+};
```

Comment:
I was going on that direction also with the idea of using components and #refs, but decided to keep it simple, since it was my first run with swagger-jsdocs. We can still do that if time allows.

Reply:
Ok! It should only take a moment though (save it as swaggerOptions.js in server/config and add `module.exports = { swaggerOptions }`, then in this file just say `const { swaggerOptions } = require('../../config/swaggerOptions')`) 😎👍